### PR TITLE
Moscow Variation -> Canal Attack

### PR DESCRIPTION
**Pull request by cygnet38**
Canal Attack and Moscow Variation refer to the same thing but were being used interchangeably, chess.com doesn't have this problem.

https://www.chess.com/openings/Sicilian-Defense-Canal-Dorfman-Gambit https://www.chess.com/openings/Sicilian-Defense-Canal-Moscow-Gambit https://www.chess.com/openings/Sicilian-Defense-Canal-Attack https://www.chess.com/openings/Sicilian-Defense-Canal-Main-Line-Haag-Gambit https://www.chess.com/openings/Sicilian-Defense-Canal-Main-Line-Sokolsky-Variation

### CODE REVIEW

File: b.tsv [hunk 1]
```diff
@@ -587,10 +587,10 @@ B50	Sicilian Defense: Modern Variations, Tartakower	1. e4 c5 2. Nf3 d6 3. d4 cxd
 B50	Sicilian Defense: Wing Gambit, Deferred Variation	1. e4 c5 2. Nf3 d6 3. b4
 B51	Sicilian Defense: Canal Attack, Dorfman Gambit	1. e4 c5 2. Nf3 d6 3. Bb5+ Nc6 4. O-O Bd7 5. Qe2 g6 6. e5
 B51	Sicilian Defense: Canal Attack, Moscow Gambit	1. e4 c5 2. Nf3 d6 3. Bb5+ Nc6 4. O-O Bd7 5. c3 Nf6 6. Re1 a6 7. Bxc6 Bxc6 8. d4 Bxe4 9. Bg5
-B51	Sicilian Defense: Moscow Variation	1. e4 c5 2. Nf3 d6 3. Bb5+
+B51	Sicilian Defense: Canal Attack,	1. e4 c5 2. Nf3 d6 3. Bb5+
```
Comment (cmgchess) [inline b.tsv:590]: can you move up this line to match the naming order. also the `,` is not needed

Comment (cygnet38) [reply b.tsv]: Thank you

**Review by niklasf — CHANGES_REQUESTED** [review]
Thanks for finding this inconsistency! We had a quick poll and found that `Moscow Variation` appears to be the more widely recognized name. Let's consistently use that, instead.

**Comment by niklasf** [comment]
Thanks!

**Pull request merged**